docs/tensorboard_in_notebooks.ipynb

```diff
@@ -4,7 +4,6 @@
   "metadata": {
```

Comment:
<p>Did the behavior of <code>load_data()</code> change? If not, what is the cause of the output cell change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/2759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>No, there is no change to behavior of <code>load_data()</code>. The current notebook has <code>!pip&nbsp;install&nbsp;-q&nbsp;tf-nightly-2.0-preview</code> which is not recommended according to <a href="https://github.com/tensorflow/tensorboard/issues/2748" target="_blank">https://github.com/tensorflow/tensorboard/issues/2748</a>. I have only removed that line from the notebook. This will not create errors due to duplicate plugins. Please see this demo for reproducing the error: <a href="https://youtu.be/8mOtyX6juZQ" target="_blank">https://youtu.be/8mOtyX6juZQ</a>.</p>